resources/js/features/games/components/GameMetadata/GameReleaseDatesRow/GameReleaseDatesRow.tsx

```diff
@@ -0,0 +1,135 @@
+import dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { BaseTableCell, BaseTableRow } from '@/common/components/+vendor/BaseTable';
+import { formatGameReleasedAt } from '@/common/utils/formatGameReleasedAt';
+
+dayjs.extend(utc);
+
+interface GameReleaseDatesRowProps {
+  releases: App.Platform.Data.GameRelease[];
+}
+
+export const GameReleaseDatesRow: FC<GameReleaseDatesRowProps> = ({ releases }) => {
+  const { t } = useTranslation();
+
+  // First, deduplicate by region, keeping only the earliest release per region.
+  const dedupedReleases = deduplicateReleasesByRegion(releases);
+
+  // Then, sort by date.
+  const sortedReleases = sortReleasesByDate(dedupedReleases);
+
+  return (
+    <BaseTableRow className="first:rounded-t-lg last:rounded-b-lg">
+      <BaseTableCell className="text-right align-top">
+        {t('metaRelease', { count: sortedReleases.length })}
+      </BaseTableCell>
+
+      <BaseTableCell>
+        <span className="flex flex-col">
+          {sortedReleases.map((release) => {
+            // Treat "other" and null as "Worldwide" for now.
+            const displayRegion =
+              !release.region || release.region === 'other' || release.region === 'worldwide'
+                ? 'WW'
+                : release.region;
+
+            // Hide the region if there's only one release and it's worldwide.
+            const shouldShowRegion = !(sortedReleases.length === 1 && displayRegion === 'WW');
+
+            return (
+              <span key={release.id}>
+                {shouldShowRegion ? (
+                  <span className="mr-1.5 font-mono uppercase">{displayRegion}</span>
+                ) : null}
+                {formatGameReleasedAt(release.releasedAt, release.releasedAtGranularity)}
```

Comment:
It might be worth adding a tooltip to this. `NA`, `EU`, and `JP` are probably straightforward enough, but people may not know what `WW` or `AS` are, and `CN` might be assumed to be Canada (why isn't it `CH` like it was in https://github.com/RetroAchievements/RAWeb/pull/3527#discussion_r2098840536?).

Reply:
I agree. Tooltip added in latest.

I missed that CN/CH discrepancy in #3527. Updated in latest, and I've added a SQL command to the description to update the ~50 records impacted in the DB.

Reply:
Going to go ahead and merge this as it touches a local branch I'm working on. If you have any feedback regarding the latest commit, I'm happy to work through it post-merge.